app/models/settings.rb

```diff
@@ -44,6 +44,7 @@ def update_attributes(attributes)
 
     super(attributes)
   end
+  alias update_attributes update
```

Comment:
Do we need this? Can we just remove `update_attributes` usage?

Reply:
I don't know why I added this line. I removed it, let's see if something fails.